app/misc/authenticate.ts

```diff
@@ -107,8 +107,8 @@ function searchRepoName() {
 
 function getUserInfo(callback) {
 
-  
-  if (signedAfter === true){  // if the trys to login after clicking "continues without sign in" 
+
+  if (signedAfter === true){  // if the trys to login after clicking "continues without sign in"
```

Comment:
I know you didn't write this one but since we're working on comments this one doesn't make any sense.

Reply:
I fixed that as well, will merge now.